pagopa/io-app
# fix: [IOBP-300] Add missing bottom sheets to ID Pay CIE screens

## Short description
This PR adds missing bottom sheets to `IDPAY_CODE_DISPLAY` and `IDPAY_CODE_RENEW` screens.

## List of changes proposed in this pull request
- [ts/features/idpay/code/screens/IdPayCodeDisplayScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-300-idpay-cie-missing-bottom-sheets?expand=1#diff-d707cf1d996ad8193c4fb386857e18e3f1defbfe49e1820b361fd7cd9b357403): added `useIdPayInfoCieBottomSheet`
- [ts/features/idpay/code/screens/IdPayCodeRenewScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-300-idpay-cie-missing-bottom-sheets?expand=1#diff-5d3cac3615ea6808724da600cc5f2ef90e9d7edc7576ef59e6c3ed70e2dcf5e8): added `useIdPayInfoCieBottomSheet`
- [ts/components/screens/WizardScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-300-idpay-cie-missing-bottom-sheets?expand=1#diff-620aa154164c40e64860e6757c936488097a73b82ec7a5a21cee93c6055bc6ab): added missing bottom `VPadding` 

## How to test
Navigate to the screens and check that the bottom sheets are shown by pressing the "How it works" button.
